blocks/share-bar-block/index.story.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import {
+  withKnobs, text, boolean,
+} from '@storybook/addon-knobs';
+import { ShareBar } from './features/share-bar/default';
+
+export default {
+  title: 'Share Bar',
+  decorators: [withKnobs],
+};
+
+// look into multiple object select
+// via https://www.npmjs.com/package/@storybook/addon-knobs/v/5.3.21#options
```

Comment:
couldn't figure out how to use multiple objects select, like a big select option field. so many options, like the preexisting helper.jsx, is good